### PR TITLE
tweak(ui): sanitize secret when export yaml

### PR DESCRIPTION
**Pull request by shhdgit**
## Why are these changes needed?
Remove the secret when exporting to avoid leaking the secret key.

## Checks

- [ ] I've included any doc changes needed for https://github.com/pingcap/LinguFlow/.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: ui/src/modules/app_builder/index.tsx [hunk 1]
```diff
@@ -278,10 +279,28 @@ const BuilderMenu: React.FC<{
     importApp(config)
   }
 
+  const sanitize = (values: { [k: string]: any }) => {
+    if (!values) {
+      return
+    }
+
+    const isSerect = values.name === SECRET_NAME
```
Comment (freedeaths) [inline ui/src/modules/app_builder/index.tsx:287]: typo: isSecret
the same in the next line

Comment (shhdgit) [reply ui/src/modules/app_builder/index.tsx]: nice catch. thanks!

**Comment by shhdgit** [comment]
@freedeaths done~

**Review by freedeaths — APPROVED** [review]
LGTM

**Pull request merged**